<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>162</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting UDColor shift</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift-R</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift-G</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>50</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift-B</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxShiftR">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-2000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxShiftG">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-2000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxShiftB">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-2000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxTimeRange">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TimeRange Sec</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCheckTime">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckTime</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxShiftOffset">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-10000</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Shift offset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
